src/app/public/modules/autocomplete/autocomplete.component.html

```diff
@@ -16,29 +16,40 @@
   #resultsTemplateRef
 >
   <div
-    class="sky-autocomplete-results sky-shadow"
-    role="listbox"
-    [attr.aria-labelledby]="ariaLabelledBy"
-    [attr.id]="resultsListId"
-    #resultsRef
+    class="sky-autocomplete-results-container"
+    [ngClass]="{
+      'sky-theme-modern' : themeName === 'modern'
```

Comment:
So this is actually a limitation of our modern mixin. That mixin uses `host-context` in its selector in order to grab the `sky-theme-modern` class outside of the view encapsulation for the component. However, the results container here is placed into an overlay which due to css limitations causes `host-context` to not work on the very first DOM element in the overlay. I could have created a new css class that toggles on with the modern theme; however, I decided we already had one that would work and so it didn't hurt to just reuse the modern theme class. I also in theory could have added another DOM element but that seemed bad semantically and would cause use to rework the overlay logic a bit.

Reply:
I see. That sounds fine then. Thanks for the explanation 👍